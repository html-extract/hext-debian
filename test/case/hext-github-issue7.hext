<div>
  <h3><a @text:title /></h3>
  <+p @text:body />
</div>
