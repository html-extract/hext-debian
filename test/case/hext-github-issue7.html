<li>
  <div>
    <h5>
      Heading 1
    </h5>
  </div>
  <div>
    <h3>
      <a href="link">Link text 1</a>
    </h3>
    <p>Line 1.1</p>
    <p>Line 1.2</p>
    <p></p>
  </div>
  <div>
    <h4>
      Heading 2
    </h4>
    <p>
      Text 1
    </p>
  </div>
</li>

<li>
  <div>
    <h5>
      Heading 3
    </h5>
  </div>
  <div>
    <h3>
      <a href="link">Link text 2</a>
    </h3>
    <p></p>
    <p>Line 2.2</p>
    <p></p>
  </div>
  <div>
    <h4>
      Heading 4
    </h4>
    <p>
      Text 2
    </p>
  </div>
</li>
