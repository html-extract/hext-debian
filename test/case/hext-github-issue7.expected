{"body":["Line 1.1","Line 1.2",""],"title":"Link text 1"}
{"body":["","Line 2.2",""],"title":"Link text 2"}
